cmd/shortener/main.go

```diff
@@ -1,3 +1,64 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+	"io"
+	"log"
+	"net/http"
+)
+
+var (
+	shortURL = map[string]string{
+		"094c4130-9674-4c18-bf60-7385d7f61934": "https://practicum.yandex.ru/",
+	}
+)
+
+func createShortURL(rw http.ResponseWriter, r *http.Request) {
+	log.Println("Create short url")
+	id := uuid.New().String()
+	bodyBytes, err := io.ReadAll(r.Body)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Please add graceful shutdown

Reply:
ok

Reply:
We should not panic in the middle of the program

Reply:
ok